src/utils/formatting.ts

```diff
@@ -70,5 +70,9 @@ export const formatting = {
 
   removeSingleLineComments(s: string): string {
     return stripJsonComments(s);
+  },
+
+  splitAndTrim(ids: string): string[] {
```

Comment:
Since this is a generic function, let's rename the `ids` arg to something more generic, like `s`